lib/ui/pages_widget.dart

```diff
@@ -26,3 +28,37 @@ class PageWidget extends StatelessWidget {
     );
   }
 }
+
+class EncryptedPageWidget extends StatelessWidget {
+  final Uri path;
+
+  EncryptedPageWidget(this.path);
+
+  @override
+  Widget build(BuildContext context) {
+    //var file = File.fromUri(path);
+    // var bytes = file.readAsBytesSync();
+    //should be this one after fixing https://github.com/flutter/flutter/issues/17419
+    //
+    // var image = Image.file(
+    //    file,
+    //   height: double.infinity,
+    //    width: double.infinity,
+    // );
+    var imageData = ScanbotSdk.getImageData(path);
```

Comment:
Where is this `getImageData` coming from? I can't find it in the SDK master. @4BooM04

Reply:
its left from the first version of encryption, i'll fix it now